variables.tf

```diff
@@ -143,6 +143,12 @@ variable "enable_host_encryption" {
   default     = false
 }
 
+variable "enable_load_balancer_profile" {
```

Comment:
Could we rename this variable to `load_balancer_profile_enabled` since now in AzureRM provider we're following such naming convention? Now there're many legacy variables with `enable_xxx` name but we'd like to keep them to keep backward compatible, but for new variables we`d like to follow the latest naming convention. Thanks!